kuma/static/styles/users.scss

```diff
@@ -13,6 +13,8 @@ Profile view
 ====================================================================== */
 @import 'components/activity';
 @import 'components/banned';
+@import 'components/readable-line-length';
+@import 'components/well';
```

Comment:
I agree with @jwhitlock, could we change this to something a bit more descriptive/semantic @joshJarr?

Reply:
Sure! This component isn't a `well` anymore, will probably call it something like `alert-message`. 
For context, called it well originally because of the old bootstrap wells https://getbootstrap.com/docs/3.3/components/#wells
Turns out these have been deprecated - wells are no longer fashionable.